components/controls/BoxedTabs.tsx

```diff
@@ -71,9 +73,10 @@ const ActiveBorder = styled.div<{ active: boolean }>`
 `;
 
 export default function BoxedTabs<K>(props: BoxedTabsProps<K>) {
-  const { tabs, selected } = props;
+  const { className, tabs, selected } = props;
+
   return (
-    <TabContainer>
+    <TabContainer className={classNames(className)}>
```

Comment:
You don't need to import and use classnames for this, and write `className={className}`.

Reply:
🤦‍♂ Good point